### PR TITLE
Simplify Overview; break other information into sections

### PR DESCRIPTION
**Pull request by emoscardini**

### CODE REVIEW

File: docusaurus/docs/introduction/01-Introduction.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+The Ziti Network (Ziti) is composed of the following building
+blocks: Ziti Controller, Ziti Router, Ziti Clients. These
+components are used in conjunction to provide secure
+connectivity between two points such as a client to a server. This
+type of network is considered an overlay network because it
+provides secure connectivity on top – or “overlaying” – existing
+networking infrastructure.
+
+Here's an overview of a network topology:
+![image](_ziti-overview.svg)
```
Comment (dovholuknf) [inline docusaurus/docs/introduction/01-Introduction.md]: static content is under /static/img. This image exists at https://github.com/openziti/ziti-doc/blob/main/docusaurus/static/img/ziti-overview.svg already. I updated main so if you merge origin/main you'll pick it up and can be referenced with `image`

File: docusaurus/docs/introduction/01-Introduction.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+id: intro
+title: What is OpenZiti?
+---
+
+**OpenZiti** is a free and open source project focused on bringing Zero Trust to any application. The project provides all the pieces required to implement or integrate Zero Trust into your solutions.
```
Comment (dovholuknf) [inline docusaurus/docs/introduction/01-Introduction.md]: please find/replace `Zero Trust` with `zero trust`

**Comment by dovholuknf** [comment]
should be fine once you make those couple of changes

File: docusaurus/docs/introduction/01-Introduction.md (hunk 1 above)
Comment (emoscardini) [reply docusaurus/docs/introduction/01-Introduction.md]: done!

File: docusaurus/docs/introduction/01-Introduction.md (hunk 2 above)
Comment (emoscardini) [reply docusaurus/docs/introduction/01-Introduction.md]: done!

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**